### PR TITLE
Implement parsing of DestinationTag during Ripple account sync

### PR DESCRIPTION
**Pull request by onyb**
Reopening #328 here. Between rebases and merging of #333, something weird happened, and it seems that #328 was finally not merged to `develop`.

### CODE REVIEW

File: core/src/database/DatabaseSessionPool.hpp [hunk 1]
```diff
@@ -59,7 +59,7 @@ namespace ledger {
                 const std::string &password = ""
             );
 
-            static const int CURRENT_DATABASE_SCHEME_VERSION = 14;
+            static const int CURRENT_DATABASE_SCHEME_VERSION = 15;
```
Comment (phaazon) [inline core/src/database/DatabaseSessionPool.hpp]: @KhalilBellakrid is going to change that value in a soon-to-be-merged PR too. Which one will have the conflict?

FIGHT! 😆

File: core/src/wallet/ripple/explorers/RippleLikeBlockchainExplorer.h [hunk 2]
```diff
@@ -62,25 +62,14 @@ namespace ledger {
...
             uint64_t confirmations;
+            Option<uint64_t> destinationTag;
+
             std::vector<api::RippleLikeMemo> memos;
 
             RippleLikeBlockchainExplorerTransaction() {
                 confirmations = 0;
             }
 
-            RippleLikeBlockchainExplorerTransaction(const RippleLikeBlockchainExplorerTransaction &cpy) {
```
Comment (phaazon) [inline core/src/wallet/ripple/explorers/RippleLikeBlockchainExplorer.h]: Why is that removed?

File: core/test/integration/BaseFixture.cpp [hunk 3]
```diff
@@ -94,6 +94,12 @@ api::AccountCreationInfo XRP_KEYS_INFO(
         {hex::toByteArray("f7e8d16154d3c7cbfa2cea35aa7a6ae0c429980892cf2d6ea9e031f57f22a63d")}
 );
 
+api::AccountCreationInfo VAULT_XRP_KEYS_INFO(
+        0, {"main"}, {"44'/144'/0'"},
+        {hex::toByteArray("03432A07E9AE9D557F160D9B1856F909E421B399E12673EEE0F4045F4F7BA151CF")},
```
Comment (phaazon) [inline core/test/integration/BaseFixture.cpp]: _All your keys are belong to us_.

File: core/src/wallet/ripple/explorers/RippleLikeBlockchainExplorer.h (hunk 2 above)
Comment (onyb) [reply core/src/wallet/ripple/explorers/RippleLikeBlockchainExplorer.h]: This copy constructor was not doing anything, according to Pierre. He asked me to remove it.

File: core/test/integration/BaseFixture.cpp (hunk 3 above)
Comment (onyb) [reply core/test/integration/BaseFixture.cpp]: By the way, these are the test keys used for integration tests and QA on Vault.

File: core/src/database/DatabaseSessionPool.hpp (hunk 1 above)
Comment (onyb) [reply core/src/database/DatabaseSessionPool.hpp]: **15** has always been my favourite number, so... 😅
